### PR TITLE
 force_single_line configuration for isort

### LINKED ISSUE #189: Use force_single_line configuration for isort
force_single_line makes diffs easier to read.

As part of this, we should probably also remove `line_length=150`.

### PR DESCRIPTION
**Pull request by Shashi456**
Fixes #189.

### CODE REVIEW

**Comment by marco-c** [comment]
There are still some flake8 failures (see Travis logs)

File: autowebcompat/utils.py [hunk 1]
```diff
@@ -8,10 +8,13 @@
 import threading
 
 from PIL import Image
-import keras
-from keras.preprocessing.image import ImageDataGenerator, img_to_array, load_img
 import numpy as np
 
+import keras
```
Comment (marco-c) [inline autowebcompat/utils.py]: These imports should be in the previous section, as it's a third-party library

File: collect.py [hunk 2]
```diff
@@ -8,10 +8,12 @@
 import traceback
 
 from PIL import Image
-from selenium import webdriver
-from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
 
 from autowebcompat import utils
+from selenium import webdriver
```
Comment (marco-c) [inline collect.py]: Here too

File: label.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import argparse
 
-import cv2
 import numpy as np
 
 from autowebcompat import utils
+import cv2
```
Comment (marco-c) [inline label.py]: Here too

File: pretrain.py [hunk 4]
```diff
@@ -4,9 +4,10 @@
 import random
 from urllib.parse import urlparse
 
-from keras.callbacks import EarlyStopping, ModelCheckpoint
-
-from autowebcompat import network, utils
+from autowebcompat import network
+from autowebcompat import utils
+from keras.callbacks import EarlyStopping
```
Comment (marco-c) [inline pretrain.py]: Here too

**Comment by Shashi456** [comment]
@marco-c  i will revise the errors, i directly executed isort on my terminal and it seems to have rearranged the files in an inappropriate way
